elephant/spade.py

```diff
@@ -1568,12 +1569,13 @@ def concept_output_to_patterns(concepts, pvalue_spectrum, winlen, binsize,
         Each element of the tuple correspond to a pattern and it is itself a
         tuple consisting of:
             ((spikes in the pattern), (occurrences of the patterns))
-    pvalue_spectrum: tuple
-        Contains a tuple of signatures and the corresponding p-value
     winlen: int
         Length (in bins) of the sliding window used for the analysis
-    t_start: int
-    t_start (in bins) of the analyzed spike trains
+    pvalue_spectrum: None or tuple
+        Contains a tuple of signatures and the corresponding p-value. If equal
+        to None all the pvalues are set to -1
```

Comment:
In the comment below you write: 
> If an empty list is given in input to the pval spectrum the pvalue is set to -1 (pvalue spectrum not available)

but here you mention that all pvalues are set to -1. It doesn't match with the documentation here. I also see that the dictionary will have afterwards only one entry, `-1`
```python
output_dict['pvalue'] = -1
```